build/scripts/entrypoint-volume.sh

```diff
@@ -52,7 +52,7 @@ if [ -n "${OPENVSX_REGISTRY_URL+x}" ]; then
   # now check if it's empty, in that case we use Plugin Registry OpenVSX
   if [ -z "$OPENVSX_REGISTRY_URL" ]; then
     # remove the suffix /v3 from this variable
-    CHE_PLUGIN_REGISTRY_ROOT_URL=${CHE_PLUGIN_REGISTRY_URL%/v3}
+    CHE_PLUGIN_REGISTRY_ROOT_URL=${CHE_PLUGIN_REGISTRY_INTERNAL_URL%/v3}
```

Comment:
probably here we need two variables, one with external access, one internal

internal should go in the product.json
external in the .js file (used by browser so it needs to reach access)